lib/ret/account.ex

```diff
@@ -6,12 +6,13 @@ defmodule Ret.Account do
 
   import Canada, only: [can?: 2]
 
+  @type id :: pos_integer
   @type t :: %__MODULE__{}
 
-  @schema_prefix "ret0"
-  @primary_key {:account_id, :id, autogenerate: true}
   @account_preloads [:login, :identity]
 
+  @schema_prefix "ret0"
```

Comment:
Does swapping the order of these lines actually have an impact or was it just shuffled in the process of doing other things?

Reply:
Or was it just that you actually have your editor correctly configured with elixir format and whoever edited before did not 😄

Reply:
`@schema_prefix` and `@primary_key` are options read by the code injected via `use Ecto.Schema`, so I put them right next to the `Ecto.Schema` DSL. `@account_preloads` is merely used as a constant so I left it on its own.

To answer your question, the order of those two doesn’t matter, I just oriented them such that the higher-level concern (tenancy) preceded the lower-level one (primary key name).